.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "openvr"]
 	path = openvr
-	url = https://github.com/ValveSoftware/openvr.git
+	url = https://github.com/ValveSoftware/openvr
```

Comment:
I had some issues with submodule when I first created my fork.  I had to delete it.  I added it back with the git submodule add.  I will change this back to .git though as I think its probably something on my end that wasn't correct.